src/github.com/getlantern/flashlight/genconfig/README.md

```diff
@@ -0,0 +1,83 @@
+# genconfig
+
+**Generate the YAML file and embedded Go datastructures containing the network configuration for the Lantern client**
+
+
+## How it works
+
+Config generation currently follows this approach:
+
+1. Build a *model*, a complete list of working chained servers, masquerades and proxied sites.
+
+1. Fill the Go code templates (*.go), based on the model.
+
+1. Fill the YAML template, based on the model. The YAML template already has filled in the LFS round-robin (flashlight servers for domain fronting).
+
+## Using genconfig
+
+### Setup
+
+You need the s3cmd tool installed and set up.  To install on Ubuntu:
```

Comment:
I don't think s3cmd is used for anything anymore.  We used to upload the configuration to S3, but we're using redis instead now.